third_party/papi_sde/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+message(STATUS "Configuring and building PAPI-SDE")
+set(TPL_PAPI_PATH "${PROJECT_BINARY_DIR}/third_party/papi_sde/src")
+ginkgo_load_and_configure_package(papi_external
+    "https://bitbucket.org/terry_cojean/papi/get/77cdd0ba8db98d86c1459dd5f55013aba242d5d5.tar.gz"
+    "SHA1=540c18a14eeafb83cd60cbbf0a96706111dbff3b"
+    "${TPL_PAPI_PATH}/src"
```

Comment:
the working directory will be in `${PROJECT_BINARY_DIR}/third_party/papi_sde/src/src`. is it supposed to?